docs/source/conf.py

```diff
@@ -160,18 +160,39 @@ def no_doc_run(self):
 # "pattern" should be replaced by "repl" in the signature of all functions in
 # "class_name"
 class_specific_replacements = {
-    "RowActionBMat8": [(r"\bBMat8\b", "Element")],
+    "RightActionPPerm16List": [
+        ("libsemigroups::PPerm<16ul, unsigned char>", "Element"),
+        ("libsemigroups::Element", "Element"),
+    ],
     "StaticTransf16": [
         ("StaticTransf16", "Transf"),
         ("PTransfBase16", "Transf"),
+        (
+            "class_<libsemigroups::Transf<16ul, unsigned char>, "
+            "libsemigroups::PTransfBase<unsigned char, "
+            "std::__1::array<unsigned char, 16ul>>>",
+            "Transf",
```

Comment:
Since the `Transf` documentation is generated by the `StaticTransf16` class, if you want the links in the signatures to work, this replacement needs to be
`("cpp_nonesense", "StaticTransf16")` . Then, `StaticTransf16` gets converted to `Tranf` by the replacement you've added in etc/replace-strings-in-doc.py.

In general, whilst developing, you may find it easier to compile the doc with `(cd docs && make html)`. This does all of the steps apart from the replacements made in etc/replace-strings-in-doc.py. To me, that makes it easier to see what keywords you need to use to make the links work correctly.

Reply:
This is also the case for `Perm` vs `StaticPerm16` and `PPerm` vs `StaticPPerm16`.

Reply:
Aha, that makes senses, I've fixed them!